releasenotes-builder/src/main/java/com/github/checkstyle/Main.java

```diff
@@ -65,6 +68,24 @@ public static void main(String... args) {
                 final CliOptions cliOptions = cliProcessor.getCliOptions();
                 final Result notesBuilderResult = runGithubNotesBuilder(cliOptions);
                 errorCounter = notesBuilderResult.getErrorMessages().size();
+
+                final String version = cliOptions.getReleaseNumber();
+                final long numberOfPeriods = version.chars().filter(ch -> ch == '.').count();
```

Comment:
move all logic to  `runPostGenerationAndPublication`

code should looks like:
```
            throws IOException, TemplateException {
        runPostGeneration(releaseNotes, cliOptions);
        ArrayList<String> errors = new ArrayList<String>(); 
        errors.addAll(validateNotes(releaseNotes, cliOptions));
        errors.addAll(runPostPublication(cliOptions);
        return errors;
    }

```

to let errors be printed after releasenotes

Reply:
> errors.addAll(validateNotes(releaseNotes, cliOptions));

@romani What is validateNotes here?

Reply:
Existing method

Reply:
There is no method like validateNotes exist

Reply:
Ups, it was from method that I guide you to create. This method should keep all your logic for version mismatch

Reply:
Please check the updated code pushed and kindly guide for the same!

Reply:
Do you mean that I should make a validateNotes method like this:

```
    private static int validateNotes(int errorCounter, CliOptions cliOptions, Result notesBuilderResult) {
        final String version = cliOptions.getReleaseNumber();
        final long numberOfPeriods = version.chars().filter(ch -> ch == '.').count();

        Multimap<String, ReleaseNotesMessage> releaseNotes = notesBuilderResult.getReleaseNotes();
        if (numberOfPeriods == 1) {
            if(releaseNotes.containsKey(Constants.BREAKING_COMPATIBILITY_LABEL)
                || releaseNotes.containsKey(Constants.NEW_FEATURE_LABEL)) {
                System.out.println("Correct validation of version");
            }
        }
        else if (numberOfPeriods == 2) {
            if (!releaseNotes.containsKey(Constants.BREAKING_COMPATIBILITY_LABEL)
                || !releaseNotes.containsKey(Constants.NEW_FEATURE_LABEL)) {
                System.out.println("[ERROR] validation of version");
                errorCounter++;
            }
        }
        return errorCounter;
    }
```

Reply:
But it should return list of Errors like it done in runPostPublication